### DISCUSSION TITLE
Help: color on fzf register_ui_select items

### DISCUSSION THREAD

**Discussion by serranomorante** [ibhagwan/fzf-lua#1513] [Q&A] [answered]
Hi! I'm using the register_ui_select utility. I like it a lot. Could you refer to me any code snippet or existent example on how can I add a color to item 2 in the screenshot? Right now I'm using the hand-point-right icon, but adding a different color just to that line would be a really good visual aid to my current selected item.

Thanks!

![image](https://github.com/user-attachments/assets/1d3319f5-45ba-44fe-88cb-15d7ad34e501)

**Comment by ibhagwan** [comment, marked as answer]
You need to use terminal ansi escape sequences, you can use fzf-lua's own utils to convert neovim hl to an ansi escape sequence:
```lua
:= require("fzf-lua").utils.ansi_from_hl("IncSearch", "text")
-- First ret val should be the wrapped string, something like:
-- ^[[48;2;255;203;139m^[[38;2;1;22;39mtext^[[0m
```

**Reply by serranomorante** [reply]
Uffff, thaaanks!
![image](https://github.com/user-attachments/assets/70d96c57-8899-4f5d-8403-2ce4c36ed619)
